### PR TITLE
Rem 713

### PR DESCRIPTION
**Pull request by Ghinwa-Arab**
-New screen 'Open Inwards Transfer', displaying only inwards transfers marked as "Work in Process." On edit, it prompts an approval window to confirm key fields before finalizing and closing the transfer.
-Fix disable condition of post button in 'Inwards Transfer' screen.

### CODE REVIEW

File: src/pages/open-inward-transfers/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      flex: 1
+    }
+  ]
+
+  const editTransfer = obj => {
+    stack({
+      Component: CloseForm,
+      props: {
+        form: obj,
+        labels,
```
Comment (Alinadimmokdad) [inline src/pages/open-inward-transfers/index.js:78]: pass it obj directly , or form and use it in the form file , is it better ?

File: src/pages/inward-transfer/forms/InwardTransferForm.js [hunk 2]
```diff
@@ -166,20 +165,6 @@ export default function InwardTransferForm({ labels, recordId, access, plantId,
   const isClosed = formik.values.wip === 2
   const isPosted = formik.values.status === 3
```
Comment (Alinadimmokdad) [inline src/pages/inward-transfer/forms/InwardTransferForm.js:166]: you can remove its not used

File: src/pages/inward-transfer/forms/InwardTransferForm.js [hunk 3]
```diff
@@ -256,7 +235,7 @@ export default function InwardTransferForm({ labels, recordId, access, plantId,
       key: 'Post',
       condition: true,
       onClick: onPost,
-      disabled: !editMode || isPosted
+      disabled: formik.values.status != 4
     },
```
Comment (Alinadimmokdad) [inline src/pages/inward-transfer/forms/InwardTransferForm.js:239]: image
remove commented code please

**Comment by Alinadimmokdad** [comment]
image
decrease the height please

**Comment by imtithalhalalo** [comment]
Screenshot 2024-12-10 091314
I cannot add

**Pull request merged**